doc/ref_model/chapters/chapter05.md

```diff
@@ -137,15 +137,15 @@ This section will detail NFVI Software Profiles and associated configurations fo
 
 | .conf            | Feature                | Type   | Basic | Network Intensive |
 |------------------|------------------------|--------|-------|-------------------|
-| nfvi.com.cfg.001 | CPU allocation ratio   | value  | 4:1   | 1:1               |
+| nfvi.com.cfg.001 | CPU allocation ratio   | value  | 1:1   | 1:1               |
```

Comment:
Suggest adding a note below the table stating that this is to allow for future performance benchmarking, allowing operators to modify for actual deployments if they are willing to accept the risk of an impact on performance.

Reply:
Sounds good.